### ISSUE TITLE
A11y_.NETWPF_EditExaminer_ImmediateWindow_Resize: At 200% zoom, Immediate Window control name gets hidden.

### ISSUE THREAD

**Issue by v-navesingh** [dotnet/wpf#5038]
Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### **GitHubTags**
#A11ySev2;#A11yMAS;#A11yTCS;#A11yWCAG;#DesktopApp;#Win10;#BM-.NetCore(WPF)-Win32-July2021;.NET Core;#WCAG1.4.4;#Resize

### **Environment details:**
Application Name: .NET Core WPF
Microsoft .NET Core SDK Version 6.0.100-rc.1.21380.2
Windows Version: Windows10​

### **Repro Steps**

1. Launch VS 2019 Int Preview
2. Tab and Navigate to Sample Applications and right click on the Editing Examiner Demo and click on build.
3..Then right click on Editing Examiner Demo then Click on Debug-->Start New Instance. 
4.Edit Examiner screen should open.
5.Tab and navigate till Immediate Window Control.
6. Zoom the page to 200% and observe the Immediate Window control Name.

### **Actual Result:**
At 200% zoom, Immediate Window control name gets hidden.

### **Expected result:**
At 200% zoom, Immediate Window control name should be visible..

### **User Impact**:
Low Vision users may find difficulty in understanding what is the title of the Control.

**Comment by singhashish-wpf** [comment]
@v-navesingh I couldn't reproduce the issue. Could you please help to share the screenshot of your repro?
Also, I am zooming via windows+ '+' (Magnifier)
![image](https://user-images.githubusercontent.com/82596204/130228237-cd340fed-2ce3-4533-b821-31c619546538.png)

**Comment by v-navesingh** [comment]
> @v-navesingh I couldn't reproduce the issue. Could you please help to share the screenshot of your repro?
> Also, I am zooming via windows+ '+' (Magnifier)
> ![image](https://user-images.githubusercontent.com/82596204/130228237-cd340fed-2ce3-4533-b821-31c619546538.png)

Choosing " make everything Bigger" issue is not reproing, this bug can be closed
![Screenshot (71)](https://user-images.githubusercontent.com/87067846/131501703-2632c07e-8bc2-473b-98ff-51310e13e963.png)

**Comment by merriemcgaw** [comment]
@singhashish-wpf - I believe you can close this issue.
